### ISSUE TITLE
moa.classifiers.meta.imbalanced Tests are failing

### ISSUE THREAD

**Issue by nuwangunasekara** [Waikato/moa#260]
It looks like moa.classifiers.meta.imbalanced Tests are failing.

# How to reproduce 
## get latest MOA master  
```
(base) ng98@MacBook-Pro-2 moa % git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

(base) ng98@MacBook-Pro-2 moa % git log |head 
commit eeade05db8e929e2ddc1987b2651310c2f76a998
Author: Aljaž Osojnik <aljaz.osojnik@ijs.si>
Date:   Tue May 10 10:40:20 2022 +0200

    Random forest for iSOUP-Tree (#254)
    
    * iSOUP-Tree integration files
    
    * Added random forest class for iSOUP-Tree
```
## build MOA using 
```mvn clean package```
Attached is the build log: 
[MOA_build_with_test_errors.log](https://github.com/Waikato/moa/files/9332408/MOA_build_with_test_errors.log)

**Comment by nuwangunasekara** [comment]
Patch to fix the issue [260.txt](https://github.com/Waikato/moa/files/9387188/260.txt).

**Comment by nuwangunasekara** [comment]
Fixed via PR #261 and #263
